files/build_templates/init_cfg.json.j2

```diff
@@ -46,7 +45,7 @@
             "has_per_asic_scope": {% if feature + '@.service' in installer_services.split(' ') %}true{% else %}false{% endif %},
             "auto_restart": "{{autorestart}}",
 {%- if include_kubernetes == "y" %}
-{%- if feature in ["dhcp_relay", "lldp", "pmon", "radv", "snmp", "telemetry"] %}
+{%- if feature in ["lldp", "pmon", "radv", "snmp", "telemetry"] %}
```

Comment:
@renukamanavalan please review this change regarding dhcp_relay and kubernetes.

Reply:
How do we ensure that the feature table entry for dhcp_relay has kube as set_owner?